pyaedt/application/Analysis3D.py

```diff
@@ -1277,6 +1277,147 @@ def import_dxf(
         self.oeditor.ImportDXF(vArg1)
         return True
 
+    @pyaedt_function_handler
+    def import_gds_3d(self, gds_file, gds_number, elevation=None, thickness=None, import_method=1):  # pragma: no cover
+        """Import a GDSII file.
+
+        Parameters
+        ----------
+        gds_file : str
+            Path to the GDS file.
+        gds_number : list
+            List of layer numbers to import.
+        elevation : list, optional.
+            List of GDS layer elevations.
+            Number of elements must be the same as gds_number list, if not elevation and thickness will not be applied.
+            If elevation is ``None``, elevation and thickness will not be applied.
+            The default is ``None``.
+        thickness : list, optional
+            List of GDS layer thicknesses.
+            Number of elements must be the same as gds_number list, if not elevation and thickness will not be applied.
+            If thickness is ``None``, elevation and thickness will not be applied.
+            The default is ``None``.
+        import_method : integer, optional
+            GDSII import method.
+            Options are:
+            - ``0`` for script
+            - ``1`` for Parasolid
+
+            The default is ``1``
```

Comment:
```suggestion
        elevation : list, optional
            List of GDS layer elevations. The number of elements in the list must be
            the same as the number of elements in the list for the ``gds_number``
            parameter.  If the number is not the same, elevation and thickness are
            not applied. If ``elevation=None``, which is the default, elevation and
            thickness are not applied.
        thickness : list, optional
            List of GDS layer thicknesses. The number of elements in the list must
            be the same as the number of elements in the list for the ``gds_number``
            parameter. If the number is not the same, elevation and thickness are not
            applied. If ``thickness=None``, which is the default, elevation and thickness
            are not applied.
        import_method : integer, optional
            GDSII import method. The default is ``1``. Options are:
            
            - ``0`` for script
            - ``1`` for Parasolid
                      
```
Avoid future tense. For a list to render correctly in the doc, a blank line must precede the first list item and follow the last list item.